extra-wm/i3status-rust/spec

```diff
@@ -0,0 +1,3 @@
+VER=0.14.2
+SRCTBL="https://github.com/greshake/i3status-rust/archive/v$VER/i3status-rust-$VER.tar.gz"
```

Comment:
Please use the new [ACBS specification file format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/#spec).